src/scripts/embed/server/views/video.handlebars

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
```

Comment:
This is the view template for our embedded video; please let me know if anything looks wrong/missing/etc.